<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sysManage</class>
 <widget class="QWidget" name="sysManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#sysManage { 
border-image: url(:/res/bg_system.png);} 
#sysManage * { 
border-image:url(); 
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>   设备状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>   密码管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restartButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41);
background-color: rgb(252, 233, 79);</string>
   </property>
   <property name="text">
    <string>重启</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>660</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(70, 173, 223);</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="devStatusTableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>482</width>
     <height>200</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>日志管理</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="devLogTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>42</y>
     <width>713</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="yearLabel">
   <property name="geometry">
    <rect>
     <x>835</x>
     <y>50</y>
     <width>85</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="LastYearButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>50</y>
     <width>75</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image: url(:/res/time_dec.png);

}

QPushButton:pressed{
border-image: url(:/res/time_dec_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="NextYearButton">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>50</y>
     <width>75</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image: url(:/res/time_add.png);

}

QPushButton:pressed{
border-image: url(:/res/time_add_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="monthLabel">
   <property name="geometry">
    <rect>
     <x>835</x>
     <y>100</y>
     <width>85</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="LastMonButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>100</y>
     <width>75</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image: url(:/res/time_dec.png);

}

QPushButton:pressed{
border-image: url(:/res/time_dec_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="NextMonButton">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>100</y>
     <width>75</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image: url(:/res/time_add.png);

}

QPushButton:pressed{
border-image: url(:/res/time_add_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dayLabel">
   <property name="geometry">
    <rect>
     <x>835</x>
     <y>150</y>
     <width>85</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="LastDayButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>150</y>
     <width>75</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image: url(:/res/time_dec.png);

}

QPushButton:pressed{
border-image: url(:/res/time_dec_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="NextDayButton">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>150</y>
     <width>75</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image: url(:/res/time_add.png);

}

QPushButton:pressed{
border-image: url(:/res/time_add_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="searchSystermLogButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>200</y>
     <width>108</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(70, 173, 223);</string>
   </property>
   <property name="text">
    <string>查看系统日志</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchWorkLogButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>200</y>
     <width>108</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(70, 173, 223);</string>
   </property>
   <property name="text">
    <string>查看操作日志</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LastPageButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>260</y>
     <width>108</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(70, 173, 223);</string>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NextPageButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>260</y>
     <width>108</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(70, 173, 223);</string>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
